corehq/apps/userreports/views.py

```diff
@@ -358,7 +358,7 @@ def page_context(self):
             'has_report_builder_access': has_report_builder_access(self.request),
             'at_report_limit': num_builder_reports >= max_allowed_reports,
             'max_allowed_reports': max_allowed_reports,
-            'support_email': settings.SUPPORT_EMAIL,
+            'support_email': settings.BILLING_EMAIL,
```

Comment:
How about renaming? It's nitpicky, but it looks like report builder has a handful of email references, some to sales and some to support, so it'd be good to be clear.